src/System.Security.Cryptography.Cng/tests/ImportExportTests.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public static class ImportExportTests
+    {
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public static void TestImportExport1()
+        {
+
+            byte[] reExported;
```

Comment:
Final PR moves reExported back into using block. Do you have any other blocking issues?

From: Stephen Toub [mailto:notifications@github.com]
Sent: Tuesday, August 4, 2015 11:07 AM
To: dotnet/corefx
Cc: Atsushi Kanamori
Subject: Re: [corefx] Implement System.Security.Cryptography.CngKey (#2615)


In src/System.Security.Cryptography.Cng/tests/ImportExportTests.cs<https://github.com/dotnet/corefx/pull/2615#discussion_r36220980>:

> +// Licensed under the MIT license. See LICENSE file in the project root for full license information.

> +

> +using Microsoft.Win32.SafeHandles;

> +using Test.Cryptography;

> +using Xunit;

> +

> +namespace System.Security.Cryptography.Cng.Tests

> +{

> +    public static class ImportExportTests

> +    {

> +        [Fact]

> +        [PlatformSpecific(PlatformID.Windows)]

> +        public static void TestImportExport1()

> +        {

> +

> +            byte[] reExported;

Nit: why is this declared all the way up here? Couldn't the assert be done inside the using as well?

—
Reply to this email directly or view it on GitHub<https://github.com/dotnet/corefx/pull/2615/files#r36220980>.